drivers/network/virtio/eth_driver.mk

```diff
@@ -25,6 +25,6 @@ eth_driver.elf: virtio/ethernet.o
 
 virtio/ethernet.o: ${ETHERNET_DRIVER_DIR}/ethernet.c ${CHECK_NETDRV_FLAGS}
 	mkdir -p virtio
-	${CC} -c ${CFLAGS} ${CFLAGS_network} -I ${ETHERNET_DRIVER} -MF virtio/ethernet.d -o $@ $<
```

Comment:
If I'm understanding correctly the dependency files are created by the included of `eth_driver.mk` (for example the echo server) and so we don't need to do it explicitly here.